src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/Manifest.cs

```diff
@@ -12,6 +12,7 @@
 using NuGet.Packaging.Core;
 using NuGet.Packaging.PackageCreation.Resources;
 using NuGet.Packaging.Xml;
+using static NuGet.Shared.XmlUtility;
```

Comment:
as mentioned in another file, I'm not a fan of this, but it's very opinionated and not part of our coding guidelines.